src/Uno.Sdk/packages.json

```diff
@@ -115,7 +115,11 @@
   },
   {
     "group": "SvgSkia",
-    "version": "3.0.6",
+    // Pinned to 3.0.3 due to missing ISvgAssetLoader APIs in Uno’s SkiaAssetLoader
+    // Tracking issue: https://github.com/unoplatform/uno/issues/21378
+    // Note: Svg.Skia is currently excluded from automatic updates by the updater
+    //       (see details in https://github.com/unoplatform/uno.templates/pull/1752)
+    "version": "3.0.3",
```

Comment:
Pinning needs also to be done as an exclusion in the updater code, in the same way we're preventing some other packages to be updated.

Reply:
Exclusion-related PR for the updater: https://github.com/unoplatform/uno.templates/pull/1752

Reply:
Eclusion already in place for the 6.2 branch now (with https://github.com/unoplatform/uno.templates/pull/1759).

https://github.com/unoplatform/uno.templates/actions/runs/17463247078/job/49592697985#step:3:204
<img width="1606" height="1270" alt="image" src="https://github.com/user-attachments/assets/da2a0230-e9f9-44c0-82d4-f1d62a6fc722" />